appveyor.yml

```diff
@@ -0,0 +1,8 @@
+version: 1.0.{build}
+image: Visual Studio 2017
```

Comment:
Actually it should be 2015. Sorry, my fault... I exported the YAML from AppVeyor and forgot to change it.